### PR TITLE
Added missing cast when getting player sides

### PR DESCRIPTION
**Pull request by StrangeOne101**
## Fixes
- Fixed missing cast when converting to radians.

### CODE REVIEW

**Comment by DomiRusz24** [comment]
I agree with this message

File: src/com/projectkorra/projectkorra/GeneralMethods.java [hunk 1]
```diff
@@ -848,7 +848,7 @@ public static String getLastUsedAbility(final Player player, final boolean check
 	 * @return the location of the distance to the left
 	 */
 	public static Location getLeftSide(final Location location, final double distance) {
-		final float angle = Math.toRadians(location.getYaw());
+		final float angle = (float) Math.toRadians(location.getYaw());
```
Comment (DomiRusz24) [inline src/com/projectkorra/projectkorra/GeneralMethods.java:851]: Very smart and concise code, it actually compiles. Bravo!

File: src/com/projectkorra/projectkorra/GeneralMethods.java [hunk 2]
```diff
@@ -898,7 +898,7 @@ public static Location getPointOnLine(final Location origin, final Location targ
 	 * @return the location of the distance to the right
 	 */
 	public static Location getRightSide(final Location location, final double distance) {
-		final float angle = Math.toRadians(location.getYaw());
+		final float angle = (float) Math.toRadians(location.getYaw());
```
Comment (DomiRusz24) [inline src/com/projectkorra/projectkorra/GeneralMethods.java:901]: Same here, good work!

**Review by DomiRusz24 — APPROVED** [review]
I approve of this message

**Review by ChristopherWMM — APPROVED** [review]

**Review by Varhagna — APPROVED** [review]

**Pull request merged**